### DISCUSSION TITLE
2021-08-18

### DISCUSSION THREAD

**Discussion by kenyonja** [Dash-Industry-Forum/webRTC#9] [General]
Minutes
- Review minutes from last meeting: https://github.com/Dash-Industry-Forum/webRTC/discussions/8
<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/tsto/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/tsto/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

Name | Attendee Email | Join Time | Leave Time | Attendance Duration
-- | -- | -- | -- | --
Thomas Stockhammer | tsto@qti.qualcomm.com | 2021-08-18 06:58:00 | 2021-08-18 11:31:54 | 274 mins
Kashyap Kammachi Sreedhar | kashyap.kammachi-sreedhar@nokia.com | 2021-08-18 07:00:40 | 2021-08-18 08:29:07 | 89 mins
Iraj | irajs@live.com | 2021-08-18 07:25:57 | 2021-08-18 08:53:00 | 88 mins
Phillip Maness Xperi | phillip.maness@xperi.com | 2021-08-18 06:59:22 | 2021-08-18 07:46:46 | 48 mins
Ali C. Begen | q@q.com | 2021-08-18 07:00:48 | 2021-08-18 07:46:52 | 47 mins
Peter Chave | pchave@akamai.com | 2021-08-18 07:00:35 | 2021-08-18 07:46:49 | 47 mins
Richard Doherty | richard.doherty@dolby.com | 2021-08-18 07:00:21 | 2021-08-18 07:46:56 | 47 mins
Emre Aksu | emre.aksu@nokia.com | 2021-08-18 07:01:20 | 2021-08-18 07:46:46 | 46 mins
Louay Bassbouss Fraunhofer FOKUS | louay.bassbouss@fokus.fraunhofer.de | 2021-08-18 07:01:41 | 2021-08-18 07:46:50 | 46 mins
Lukasz Kondrad | lukasz.kondrad@nokia.ocm | 2021-08-18 07:01:01 | 2021-08-18 07:46:58 | 46 mins
Alexander Solonskycastlabs | alexander.solonsky@castlabs.com | 2021-08-18 07:02:47 | 2021-08-18 07:46:53 | 45 mins
Mike English | englishm@llnw.com | 2021-08-18 07:02:30 | 2021-08-18 07:46:52 | 45 mins
Ofer Shem Tov | ost@verizonmedia.com | 2021-08-18 07:01:57 | 2021-08-18 07:46:51 | 45 mins
Sassan | s.pejhan@ateme.com | 2021-08-18 07:01:58 | 2021-08-18 07:46:46 | 45 mins
Nate Burr | nate.burr@verizondigitalmedia.com | 2021-08-18 07:01:36 | 2021-08-18 07:45:00 | 44 mins
Piers O'Hanlon | piers.ohanlon@bbc.co.uk | 2021-08-18 07:03:14 | 2021-08-18 07:46:50 | 44 mins
Giri Mandyam | mandyam@qti.qualcomm.com | 2021-08-18 07:04:56 | 2021-08-18 07:46:59 | 43 mins
Kashyap | kashyap.kammachi-sreedhar@nokia.com | 2021-08-18 07:05:02 | 2021-08-18 07:47:07 | 43 mins
Andy Rosen - SequenceKey.com | arosen@sequencekey.com | 2021-08-18 07:12:33 | 2021-08-18 07:47:02 | 35 mins
Yasser Syed- Comcast | yasser_syed@comcast.com | 2021-08-18 07:18:25 | 2021-08-18 07:46:57 | 29 mins
Kashyap | kashyap.kammachi-sreedhar@nokia.com | 2021-08-18 07:47:40 | 2021-08-18 07:47:51 | 1 mins
Kashyap Kammachi Sreedhar | kashyap.kammachi-sreedhar@nokia.com | 2021-08-18 06:58:16 | 2021-08-18 06:58:28 | 1 mins

</body>

</html>

- Review draft agenda
- Action item review:
   - Update and refine the scope based on the discussion
   - Continue to refine the report, adding information - In progress

- Document review
   - Link: https://docs.google.com/document/d/1vwtJCgE95d2bPthwfKlvGxPiElgqJQZf7rtz_g_ZgZw/edit?usp=sharing
   - Review updates to content in clauses 6, 7, and 8
   - Discuss Clause 8, discuss usage of DASH MPDs for WebRTC

**Comment by acbegen** [comment]
Based on the agreement, I cleaned up the document and actually produced a new version. We might want to continue with this document (in that case, we should update readme.md):
https://docs.google.com/document/d/1iaWO_oAjUH-Mvrj4bzzD06YENu48RXaZx5DrJOyoT6s/edit?usp=sharing
